src/js/helpers/aboutTheDataHelper.js

```diff
@@ -3,31 +3,20 @@
  * Created by Jonathan Hill 11/20/20
  */
 
+import { stringify } from 'query-string';
```

Comment:
we should remove this from out package.json file, node has a queryString module. 

https://nodejs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options